tools/slintpad/package.json

```diff
@@ -6,8 +6,8 @@
     "main": "index.js",
     "scripts": {
         "build": "npm run clean && npx vite build",
-        "build:wasm_lsp": "wasm-pack build --dev --target web ../lsp -- --no-default-features --features backend-winit,renderer-femtovg,preview",
-        "build:wasm_lsp-release": "wasm-pack build --release --target web ../lsp -- --no-default-features --features backend-winit,renderer-femtovg,preview",
+        "build:wasm_lsp": "cross-env SLINT_ENABLE_EXPERIMENTAL_FEATURES=1 wasm-pack build --dev --target web ../lsp -- --no-default-features --features backend-winit,renderer-femtovg,preview",
+        "build:wasm_lsp-release": "cross-env SLINT_ENABLE_EXPERIMENTAL_FEATURES=1 wasm-pack build --release --target web ../lsp -- --no-default-features --features backend-winit,renderer-femtovg,preview",
```

Comment:
That's not a good way to set the env variable becuase that means the lsp don't build stand alone with cargo install.
Have you tried to set the env variable in the lsp/build.rs with the correct printf!() incentation?